packages/cli/src/databases/repositories/test-run.repository.ee.ts

```diff
@@ -26,4 +28,18 @@ export class TestRunRepository extends Repository<TestRun> {
 	public async markAsCompleted(id: string, metrics: AggregatedTestRunMetrics) {
 		return await this.update(id, { status: 'completed', completedAt: new Date(), metrics });
 	}
+
+	public async getMany(testDefinitionId: string, options: ListQuery.Options) {
+		const findManyOptions: FindManyOptions<TestRun> = {
+			where: { testDefinition: { id: testDefinitionId } },
+			order: { createdAt: 'DESC' },
+		};
+
+		if (options?.take) {
```

Comment:
Is it possible to define only `skip` but no `take`? Or is that a BadRequest?

Reply:
This is being checked in a common pagination middleware: https://github.com/n8n-io/n8n/blob/40a41dd19283617e49b0156acdbf24b14a997fd0/packages/cli/src/middlewares/list-query/pagination.ts#L19